pkg/osquery/runtime/runtime.go

```diff
@@ -795,6 +809,11 @@ func (r *Runner) launchOsqueryInstance() error {
 		return errors.Wrap(err, "could not create an extension client")
 	}
 
+	err = o.stats.Connected(o)
+	if err != nil {
+		level.Info(o.logger).Log("msg", fmt.Sprint("osquery instance history error: ", err.Error()))
+	}
```

Comment:
```suggestion
	if err := o.stats.Connected(o); err != nil {
		level.Info(o.logger).Log("msg", "osquery instance history",  "error", err)
	}
```

A couple of nits... For something like this, I tend to favor merging the if and err assignment. I like the scoping. And for logging I tend to favor structured logging and not string manipulation,

Reply:
updated